releasenotes/notes/proc-batch-size-844ecfed36a30c47.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixes an issue where `process_config.max_per_message` and `process_config.max_message_bytes`
+    were ignored when set larger than the default values.
```

Comment:
This could potentially be interpreted as saying that arbitrarily large values are now accepted for these configs - worth clarifying that there is now a larger limit on the accepted values?